controls/cis_ocp_1_3_0.yml

```diff
@@ -1,10 +1,9 @@
 policy: CIS RedHat OpenShift Container Platform Benchmark
 title: CIS RedHat OpenShift Container Platform Benchmark
-id: PLACEHOLDER
+id: cis_ocp_1.3.0
```

Comment:
We need to discuss how we want to handle benchmark versions for the Compliance Operator.

Ideally, if we're introducing more machinery to help automate profile maintenance, it would be nice to version each profile in a way that users see so that they know what version the checks correspond to.

Reply:
I think exposing the name in the version is the right step. How we deprecate the older versions is up for discussion and mostly a downstream thing to decide.